tests/gdx-tests/src/com/badlogic/gdx/tests/BitmapFontTest.java

```diff
@@ -115,7 +115,7 @@ public void render () {
 			text = "Another font wrap is-sue,  this time with multiple whitespace characters.";
 			// text = "test with AGWlWi AGWlWi issue";
 			text = "AA BB \nEE"; // When wrapping after BB, there should not be a blank line before EE.
```

Comment:
The first assignment to `text` is ignored because it is reassigned on the next line.